platform/ios/src/UIImage+MGLAdditions.h

```diff
@@ -10,6 +10,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (std::unique_ptr<mbgl::style::Image>)mgl_styleImageWithIdentifier:(NSString *)identifier;
 
+- (mbgl::PremultipliedImage)mgl_PremultipliedImage;
```

Comment:
Nit: `mgl_premultipliedImage` would be a more natural spelling for this method name.